docs/api/graph/heatmap.md

```diff
@@ -0,0 +1,90 @@
+Atlas primarily supports visualizing data in line charts. As of 1.8, Atlas can also visualize
+via heatmaps using the [`:heatmap`](../../api/graph/line-styles.md#heat-map) line style.
+The graph area is broken up into a series of cells and a count for each cell is incremented when 
+a measurement falls within the cells boundaries. Colors or shades of colors then fill in cells 
+based on the final count.
+
+## Percentiles
+Heat maps are particularly useful on top of
+[percentile](../../spectator/patterns/percentile-timer.md) metrics to analyze the entire
+measurement range. 
+
+**Note** the query must be setup properly for percentile metrics. Instead of using the 
+`:percentile` operator, the query must group on the `percentile` tag via `(,percentile,),:by`.
+Using the [log linear](axis-scale.md#log-linear) scale will help to highlight clustered
+regions of measurements via `&scale=log-linear`.
+The example also uses data not available in the demo Atlas instance.
+
+@@@ atlas-uri { hilite=:heatmap }
+/api/v1/graph?e=2012-01-01T00:00&no_legend=1&q=name,requestLatency,:eq,(,percentile,),:by,:heatmap&scale=log-linear
+@@@
+
+![ptile-heatmap](../../images/heatmap_timer2_log_linear.png)
+
+## Bounds
+
+The `&heatmap_l=` and `&heatmap_u` parameters can be used to narrow the range of cells 
+displayed in a heatmap. Heatmap bounds act on the _count_ of measurements in a cell and the
+palette colors or shades chosen. Bounds may have the effect of omitting some cells that are
+above or below the configured boundaries.
```

Comment:
The normal bounds for the value axis may omit some cells, but the heatmap bounds just impact the color scale. The set of cells will not change.

Reply:
ack. In my version I was re-calculating the min and max after bound filtering so it could would tweak the axis scale. I'll fix it the doc.